src/Dfe.EarlyYearsQualification.Content/RichTextParsing/Helpers/NestedContentHelper.cs

```diff
@@ -1,9 +1,9 @@
 using System.Text;
 using Contentful.Core.Models;
-using Dfe.EarlyYearsQualification.Content.Renderers.GovUk;
-using ParagraphRenderer = Dfe.EarlyYearsQualification.Content.Renderers.GovUk.ParagraphRenderer;
+using Dfe.EarlyYearsQualification.Content.RichTextParsing.Renderers;
+using ParagraphRenderer = Dfe.EarlyYearsQualification.Content.RichTextParsing.Renderers.ParagraphRenderer;
```

Comment:
Same as above, could be replaced by specifying `RichTextParsing.Renderers.ParagraphRenderer` instead?